src/tiled/mainwindow.cpp

```diff
@@ -1953,7 +1960,23 @@ void MainWindow::editTilesetProperties()
     tilesetDocument->setCurrentObject(tilesetDocument->tileset().data());
     emit tilesetDocument->editCurrentObject();
 }
-
+void MainWindow::editWorldProperties()
+{
+    if (WorldManager::instance().worlds().empty())
+        return;
+    QSharedPointer<WorldDocument> world;
+    Document *currentDocument = DocumentManager::instance()->currentDocument();
+    if (currentDocument && currentDocument->type() == Document::MapDocumentType)
+        world = WorldManager::instance().worldForMap(currentDocument->fileName());
+    if (!world)
+        world = WorldManager::instance().worlds().first();
+
+    if (WorldPropertiesDialog(world, this).exec() == QDialog::Accepted) {
+        QString saveError;
+        if (!world->save(world->fileName(), &saveError)) // my hero ...
```

Comment:
What does "// my hero ..." mean?

Reply:
Sorry just a joke.  "Save the world". I'll remove it 😛